src/pages/LogIn/LoginForm.tsx

```diff
@@ -25,7 +25,7 @@ export function LoginForm() {
     }
     dispatch(
       logIn({
-        username: email.trim(),
+        username: email,
```

Comment:
Why do we need to remove the trimming?
It just removes white spaces at the end or start of the email (like `str.strip`) in python